lib/hooks/useFacets/useFacets.js

```diff
@@ -35,6 +38,15 @@ a different query is generated because the `selectedBrowseResult` parameter is r
 change made is to the `selectedBrowseResult`, all open facets need to be fetched.
 */
 
+const FACETS_FETCH_ALL = [
+  FACETS.CONTRIBUTORS_HELD_BY,
+  FACETS.CONTRIBUTORS_SHARED,
+  FACETS.SUBJECTS_HELD_BY,
+  FACETS.SUBJECTS_SHARED,
+  FACETS.SUBJECT_SOURCE,
+  FACETS.SUBJECT_TYPE,
+];
```

Comment:
Let's rename this variable, in this list we only want to have facets that use the multiselect component and require more than 6 options.

The problem is that previously we didn't have the multiselect components used for facets that require more than 6 options.

Only newly added facets (`FACETS.SUBJECT_SOURCE` and `FACETS.SUBJECT_TYPE`) require this change.

Shared facet only has two options (maybe even supports +More button).

Held by already supports fetching all options (it has +More button).